packages/contracts/src/RelayHub.sol

```diff
@@ -89,59 +97,72 @@ contract RelayHub is IRelayHub, Ownable, ERC165 {
         setConfiguration(_config);
     }
 
+    /// @inheritdoc IRelayHub
     function getCreationBlock() external override view returns (uint256){
         return creationBlock;
     }
 
+    /// @inheritdoc IRelayHub
     function getDeprecationTime() external override view returns (uint256) {
         return deprecationTime;
     }
 
+    /// @inheritdoc IRelayHub
     function getStakeManager() external override view returns (IStakeManager) {
         return stakeManager;
     }
 
+    /// @inheritdoc IRelayHub
     function getPenalizer() external override view returns (address) {
         return penalizer;
     }
 
+    /// @inheritdoc IRelayHub
     function getBatchGateway() external override view returns (address) {
         return batchGateway;
     }
 
+    /// @inheritdoc IRelayHub
     function getRelayRegistrar() external override view returns (address) {
         return relayRegistrar;
     }
 
+    /// @inheritdoc IRelayHub
     function getMinimumStakePerToken(IERC20 token) external override view returns (uint256) {
         return minimumStakePerToken[token];
     }
 
+    /// @inheritdoc IRelayHub
     function getWorkerManager(address worker) external override view returns (address) {
         return workerToManager[worker];
     }
 
+    /// @inheritdoc IRelayHub
     function getWorkerCount(address manager) external override view returns (uint256) {
         return workerCount[manager];
     }
 
+    /// @inheritdoc IERC165
     function supportsInterface(bytes4 interfaceId) public view virtual override(IERC165, ERC165) returns (bool) {
         return interfaceId == type(IRelayHub).interfaceId ||
             interfaceId == type(Ownable).interfaceId ||
             super.supportsInterface(interfaceId);
     }
 
+    // TODO: make registrar immutable
```

Comment:
todo: should inherit Registrar and avoid this setter at all (relevant to this commit just the same..)

Reply:
Is currently working on it in a separate PR